test/unit/test_sap_adt_messageclass.py

```
@@ -0,0 +1,288 @@
+    def test_set_message_unlock_on_error(self):
+        lock_msg_resp = Response(
+            text=LOCK_RESPONSE_XML, status_code=200,
+            headers={'Content-Type': 'application/vnd.sap.as+xml; charset=utf-8; dataname=com.sap.adt.lock.Result'}
+        )
+        lock_class_resp = Response(
+            text='<?xml version="1.0" encoding="UTF-8"?><asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0"><asx:values><DATA><LOCK_HANDLE>CLASS_LOCK</LOCK_HANDLE></DATA></asx:values></asx:abap>',
+            status_code=200,
+            headers={'Content-Type': 'application/vnd.sap.as+xml; charset=utf-8; dataname=com.sap.adt.lock.Result'}
+        )
+        get_resp = Response(text=MESSAGE_CLASS_ADT_GET_RESPONSE_EMPTY_XML, status_code=200, headers={})
+        unlock_class_resp = Response(text='', status_code=200, headers={})
```

Flake8 (7.3.0)

[error] local variable 'unlock_class_resp' is assigned to but never used

(F841)